frontend/packages/ux-editor/src/containers/FormDesigner.tsx

```diff
@@ -112,10 +113,10 @@ export const FormDesigner = ({ selectedLayout, selectedLayoutSet }: FormDesigner
       <DndProvider backend={HTML5Backend}>
         <div className={classes.root}>
           <div className={classes.container}>
-            <LeftMenu className={classes.leftContent + ' ' + classes.item} />
+            <LeftMenu className={cn(classes.leftContent, classes.item)} />
             <FormContextProvider>
-              <DesignView className={classes.mainContent + ' ' + classes.item} />
-              <RightMenu className={classes.rightContent + ' ' + classes.item} />
+              <DesignView className={cn(classes.mainContent, classes.item)} />
+              <RightMenu className={cn(classes.rightContent, classes.item)} />
```

Comment:
Jeg føler ikke at det er nødvendig å importere en funksjon bare for å sette inn et mellomrom mellom to klasser, men det spiller ikke stor rolle. Selv bruker jeg denne bare når noen av klassene kan være `undefined`.